Tests/test_file_tiff_metadata.py

```diff
@@ -155,7 +155,7 @@ def test_iccprofile(self):
 
         im.save(out)
         reloaded = Image.open(out)
-        self.assert_(type(im.info['icc_profile']) is not type(tuple))
+        self.assertTrue(type(im.info['icc_profile']) is not type(tuple))
```

Comment:
It's @wiredfool's test originally, PR https://github.com/python-pillow/Pillow/pull/1465 to fix https://github.com/python-pillow/Pillow/issues/1462:

 * In 2.9.0 `im.info['icc_profile']` returned a bytestring.
 * In 3.0.0 `im.info['icc_profile']` returned a tuple.

There's at least a bug in the test:

```python
    self.assert_(type(im.info['icc_profile']) is not type(tuple))
```

Because `type(tuple)` returns `<type 'type'>` and not `<type 'tuple'>`. 

`type(im.info['icc_profile']) is not type(tuple)` is always going to be True (unless we *really* break it!)


So the updated test checks it properly:
```python
self.assertNotIsInstance(im.info['icc_profile'], tuple)
```

@wiredfool Is this still a bug in the TiffTags.py or should `im.info['icc_profile']` return a tuple?